metadata/rest/instrument_queries/instrument_categories.py

```diff
@@ -0,0 +1,45 @@
+"""CherryPy Metadata Instrument Categories Class."""
+import re
+from collections import defaultdict
+from cherrypy import tools
+from metadata.orm import Instruments
+from metadata.rest.instrument_queries.query_base import QueryBase
+
+
+class InstrumentCategories(QueryBase):
+    """Calculates a set of instrument categories from the display names of EUS instruments."""
+
+    exposed = True
+
+    @staticmethod
+    def derived_instrument_categories():
+        """Extract category names from the display name strings for EUS Instruments."""
+        inst_collection = Instruments.select(
+            Instruments.id, Instruments.name, Instruments.display_name
+        )
+        category_list = []
+        categorized_instruments = defaultdict(list)
+        uncategorized_instruments = {}
+        for inst in inst_collection:
+            # print "ID => {0} / inst_name => {1} / disp_name => {2}".format(inst.id, inst.name, inst.display_name)
+            match = re.match(r'^(.+?):\s+(.+)', inst.name)
```

Comment:
This is because EUS is stupid, we should put this logic into the EMSL EUS Sync script to create appropriate instrument groups then use those to group instruments.